aries.md

```diff
@@ -0,0 +1,23 @@
+* Connection:
+    * Inviter
+        * [API](./libvcx/api/connection.rs) 
+        * [State diagram](docs/states/aries/connection-inviter.puml) 
+    * Invitee
+        * [API](./libvcx/api/connection.rs) 
+        * [State diagram](docs/states/aries/connection-invitee.puml) 
+
+* Credential Issuance:
+    * Issuer
+        * [API](./libvcx/api/issuer_credential.rs) 
+        * [State diagram](docs/states/aries/issuer-credential.puml) 
+    * Holder
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/credential.rs) 
+        * [State diagram](docs/states/aries/credential.puml) 
+
+* Credential Presentation:
+    * Verifier
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/proof.rs) 
+        * [State diagram](docs/states/aries/proof.puml) 
+    * Prover
+        * [API](https://github.com/hyperledger/indy-sdk/tree/master/vcx/libvcx/api/disclosed_proof.rs) 
+        * [State diagram](docs/states/aries/disclosed-proof.puml) 
```

Comment:
We should consider generating `.png`s from the `.puml`s and updating them if necessary. Or removing them althogher, maybe links to the RFCs are sufficient? There is an overhead associated with upkeeping the diagrams, and they also change with versions.

Reply:
I don't want to get into that now, just wanted to move this to reduce noise in readme.md

Reply:
https://github.com/AbsaOSS/libvcx/issues/43